cli/internal/cmd/iamcreate.go

```diff
@@ -59,8 +59,8 @@ func newIAMCreateCmd() *cobra.Command {
 	}
 
 	cmd.PersistentFlags().BoolP("yes", "y", false, "create the IAM configuration without further confirmation")
-	cmd.PersistentFlags().Bool("generate-config", false, "automatically generate a configuration file and fill in the required fields")
-	cmd.PersistentFlags().StringP("kubernetes", "k", semver.MajorMinor(config.Default().KubernetesVersion), "Kubernetes version to use in format MAJOR.MINOR - only usable in combination with --generate-config")
+	cmd.PersistentFlags().Bool("update-config", false, "automatically update the config file with the specific IAM information")
+	cmd.PersistentFlags().StringP("kubernetes", "k", semver.MajorMinor(config.Default().KubernetesVersion), "Kubernetes version to use in format MAJOR.MINOR - only usable in combination with --update-config")
```

Comment:
just a small hint: this change removes the need for the `kubernetes` flag in `iam create`. So the flag can be remove here. :)